src/views/BusinessCase/AlternativeSolution/index.tsx

```diff
@@ -239,12 +244,12 @@ const AlternativeSolution = ({
 
         {altLetter === 'A' && !values.alternativeB && (
           <>
-            <h2 className="margin-bottom-1">Additional Alternatives</h2>
+            <h2 className="margin-bottom-1">Additional alternatives</h2>
             <HelpText>
-              If you are building a multi-year project that will require
-              significant upkeep, you may want to include more options. Keep in
-              mind that Government Off the Shelf and Commercial Off the Shelf
-              are acceptable alternatives to include.
+              If you are buillding a multi-year project that will require
+              significant upkeep, you may want to include more alternatives.
+              Keep in mind that Government off-the-shelf and Commercial
```

Comment:
This is the second time COTS and GOTS are on the page. Should these now be acronyms?

Reply:
Sai and I made updates based on the EASi Content Strategy sheet.

https://docs.google.com/spreadsheets/d/1m73GZ5CxmThy-LdvW2Xhfs-kuPS2-_BvoHQVGSjwXQI/edit#gid=1561333309&range=C58